Source/Extensions/Blazorise.TreeView/TreeView.razor.cs

```diff
@@ -131,10 +131,20 @@ public async Task Reload()
     {
         treeViewNodeStates = new();
 
-        await foreach ( var nodeState in Nodes.ToNodeStates( HasChildNodesAsync, HasChildNodes, ( node ) => ExpandedNodes?.Contains( node ) == true ) )
+        await foreach ( var nodeState in Nodes.ToNodeStates( HasChildNodesAsync, HasChildNodes, ( node ) => ExpandedNodes?.Contains( node ) == true, IsDisabled ) )
         {
             treeViewNodeStates.Add( nodeState );
         }
+
+        ////traverse nodeStateTree and fill in children for expanded nodes

```

Comment:
Was here something you tried to do and forgot? Or can we remove the commented code?